packages/eslint-plugin-office-addins/src/utils/load.ts

```diff
@@ -41,28 +41,46 @@ function composeObjectExpressionPropertyIntoString(
   return composedProperty;
 }
 
+function parseLoadStringArgument(argument: string): string[] {
+  let properties: string[] = [];
+  argument
+    .replace(/\s/g, "")
+    .split(",")
+    .forEach((property: string) => {
+      properties.push(property);
+    });
+
+  return properties;
+}
+
 export function parseLoadArguments(node: TSESTree.MemberExpression): string[] {
   node = findTopLevelExpression(node);
 
   if (
     isLoadFunction(node) &&
     node.parent?.type === TSESTree.AST_NODE_TYPES.CallExpression
   ) {
+    const argument = node.parent.arguments[0];
+    if (!argument) return [];
+
     let properties: string[] = [];
-    node.parent.arguments.forEach(
-      (argument: TSESTree.CallExpressionArgument) => {
-        if (argument.type === TSESTree.AST_NODE_TYPES.Literal) {
-          (argument.value as string)
-            .replace(/\s/g, "")
-            .split(",")
-            .forEach((property: string) => {
-              properties.push(property);
-            });
-        } else if (argument.type === TSESTree.AST_NODE_TYPES.ObjectExpression) {
-          properties.push(composeObjectExpressionPropertyIntoString(argument));
+    if (argument.type === AST_NODE_TYPES.ArrayExpression) {
+      argument.elements.forEach((args) => {
+        if (args.type === TSESTree.AST_NODE_TYPES.Literal) {
+          properties = properties.concat(
+            parseLoadStringArgument(args.value as string)
+          );
         }
```

Comment:
Can the array have the object expression in it (is it valid)?

Reply:
It is not valid. The use of object is a separate case of load. I feel we still need more tests with it, I had created a task about supporting better this case of load: https://office.visualstudio.com/OC/_workitems/edit/5422476

Reply:
So 'load([{font { size: 12} }])' isn't valid in an Office.js load call?  Seems like if it works as a direct argument in the load it could work as an element in the array . . . but I'm not the one that implemented it in Office.js.

Reply:
It is not valid. These are the parameters Load can receive (official docs):
https://docs.microsoft.com/en-us/javascript/api/excel/excel.chart?view=excel-js-preview#load_options_

Reply:
OK.

Seems like it might be nice to flag that that case in a rule (new one? existing one?) since it doesn't work, but seems like it should.  Will log a task for this.

Note: I'm noticing a lack of object style argument tests in rules that are parsing the load argument (like load-object-before-read).

Reply:
Added a comment to that task to include more tests as well.

created: https://office.visualstudio.com/OC/_workitems/edit/5461502 for the object rule